build.gradle.kts

```diff
@@ -169,7 +169,7 @@ spotless {
     java {
         licenseHeaderFile("gradle/spotless/license.java")
 
-        googleJavaFormat().aosp()
+        googleJavaFormat("1.7").aosp()
```

Comment:
Just out of curiosity does this indicate a Java version or a version of the format? (Or something totally different)

Reply:
Version of the format, newer versions require Java 11.